index.html

```diff
@@ -0,0 +1,88 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Email Template-2</title>
+  <style>
+    @import url('https://fonts.googleapis.com/css2?family=Anek+Malayalam&family=Lato:wght@400;700&family=Raleway:ital,wght@0,100;0,400;1,200&display=swap');
+
+    * {
+      margin: 0;
+      padding: 0;
+      border: 0;
+    }
+    body {
+      font-family: "Raleway", sans-serif;
+      background-color: #d9dbdb;
+      font-size: 19px;
+      max-width: 800px;
+      margin: 0 auto;
+      padding: 3%;
+    }
+
+    img {
+      max-width: 100%;
+    }
+
+    header {
+      width: 98%;
+    }
+
+    #logo {
+      max-width: 150px;
+      margin: 10px auto;
+      padding: 25px;
+    }
+
+    #wrapper {
+      background-color: #ffffff;
+    }
+
+    p {
+      margin: 5%;
+    }
+
+    .btn {
+      margin: 3% 2% 5% 5%;
+      background-color: #6cd610;
+      color: #ffffff;
+      text-decoration: none;
+      font-weight: 800;
+      padding: 15px 12px;
+      font-style: italic;
+      border-radius: 8px;
+      letter-spacing: 2px;
+    }
+
+    #contact {
+      text-align: center;
+      padding-bottom: 3%;
+      line-height: 16px;
+      font-size: 12px;
+      color: #303840;
+    }
+  </style>
+</head>
+<body>
```

Comment:
This is a good starting template but is very simple. Look at some of the emails you have received after registering to various services and get ideas on how to add more items in there.
Also, in the content part of the template, try adding some links for users to take action.

Reply:
@schohan thanks for your feedback, I will implement the changes